routes/api/piggyBanks.js

```diff
@@ -0,0 +1,11 @@
+const piggyBanksController = require("../../controllers/piggyBanksController");
```

Comment:
in my opinion if my project is using import/export in the front end i also use it in the back-end, but hey that's just me. I keep in consistent.